examples/test/Menu_test.js

```diff
@@ -0,0 +1,158 @@
+// (C) 2007-2018 GoodData Corporation
+import { Selector } from 'testcafe';
+import { config } from './utils/config';
+import { loginUsingLoginForm, waitForPivotTableStopLoading } from './utils/helpers';
+
+const totalValues = {
+    sum: ['Sum', '', '', '$1,566,007', '$150,709'],
+    max: ['Max', '', '', '$101,055', '$25,140'],
+    empty: '–'
+};
+
+const turnOnMenu = async (t) => {
+    await t.click(Selector('.s-total-preset-aggregations'));
+    await waitForPivotTableStopLoading(t);
+};
+
+const getMeasureCell = (column) => {
+    return Selector(`[col-id="a_2009_1-a_2071_1-m_${column}"] .s-pivot-table-column-header`);
+};
+
+const getMeasureGroupCell = (column) => {
+    return Selector(`[col-id="2_${column}"]`);
+};
+
+const getPivotTableFooterCell = (row, column) => {
+    return Selector(`[row-index="b-${row}"] .s-cell-${row}-${column}`);
```

Comment:
Is there an `s-` based selector instead of the attribute one? If not, would it be hard to add one?

Reply:
Same reasoning as above